src/components/lco/PageBanner.tsx

```diff
@@ -12,38 +11,45 @@ export interface LCOProfileType {
   areaIdList: string[]
   /** Official name */
   name: string
+  description: string
+  email?: string
+  facebook?: string
   /** IG Url */
-  instagram: string
+  instagram?: string
   /** Official website */
-  website: string
+  website?: string
   /** Bad hardware report form url */
   report?: string
   /** Donation url */
   donation?: string
 }
 
-const findLCOs = (
-  lcoList: LCOProfileType[],
-  currentPageAncestors: string[]
-): LCOProfileType[] => {
-  return lcoList.reduce<LCOProfileType[]>((acc, curr) => {
-    if (isMyCrag(currentPageAncestors, curr.areaIdList)) {
-      acc.push(curr)
-    }
-    return acc
-  }, [])
-}
-
-const isMyCrag = (ancestors: string[], myCrags: string[]): boolean =>
-  ancestors.some((path) => myCrags.some((myCragId) => myCragId === path))
-
-type PageBannerProps = Pick<ClimbType, 'ancestors'>
+interface PageBannerProps{orgs: OrganizationType[]}
+const getLcoList = (orgs): LCOProfileType[] => {
+  const lcoList: LCOProfileType[] = []
 
+  orgs.forEach(org => {
```

Comment:
Might simplify the code a little to do map directly rather than push into an empty array:
```
const lcoList: LCOProfileType[] = orgs.map(org => ({
  id: ..
 }))
```

Reply:
Thanks for the suggestions! Good catch on checking orgType and using map.
Yeah currently it works on FE

Reply:
Actually, since I have to change the keys of the org object, I am not sure how to do this with a map.